google.visualization/google.visualization.d.ts

```diff
@@ -364,7 +364,8 @@ declare module google {
         // https://google-developers.appspot.com/chart/interactive/docs/animation
         export interface TransitionAnimation {
             duration?: number;
-            easing?: string; // linear, in, out, inAndOut
+            easing?: string; // linear, in, out, inAndOut,
+            startup: boolean; 

```

Comment:
Sorry about that. I'll submit another pull request with the correction. Thanks for pointing that out!

Sent from my iPhone

> On Dec 10, 2015, at 7:57 AM, Masahiro Wakame <notifications@github.com> wrote:
> 
> In google.visualization/google.visualization.d.ts:
> 
> > @@ -364,7 +364,8 @@ declare module google {
> >          // https://google-developers.appspot.com/chart/interactive/docs/animation
> >          export interface TransitionAnimation {
> >              duration?: number;
> > -            easing?: string; // linear, in, out, inAndOut
> > +            easing?: string; // linear, in, out, inAndOut,
> > +            startup: boolean; 
> this property is not optional?
> 
> —
> Reply to this email directly or view it on GitHub.
>